cc_library/py-scc/src/sciler/scclib/app.py

```diff
@@ -15,12 +15,11 @@ def __init__(self, config, device):
         Initialize device with its configuration json file and python script.
         """
         self.device = device
-        self.config = json.load(config)
-        self.name = self.config.get("id")
-        self.info = self.config.get("description")
-        self.host = self.config.get("host")
-        self.port = self.config.get("port")
-        self.labels = self.config.get("labels")
+        self.name = config.get("id")
```

Comment:
why remove self?

Reply:
self.config wasn't used

Reply:
self.config wasn't used

Reply:
config should be used for checking input and output types so it should still be a self variable

Reply:
added it back